tools/test-lib

```diff
@@ -1,51 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 
```

Comment:
Any particular reason, why we should not be using python3 here?

Reply:
It's been a while since I looked into this, but my recollection was we have a lint rule against it.

For reference, currently we get:
```
$ git grep /env|grep tools|grep -v bash
tools/custom_check.py:        {'pattern': '#!/usr/bin/env python$',
tools/custom_check.py:         'description': 'Fix shebang line with proper call to /usr/bin/env for Bash path, change -x|-e switches'
tools/deploy:#!/usr/bin/env python
tools/lint:#! /usr/bin/env python
tools/provision:#!/usr/bin/env python3
tools/release-packages:#!/usr/bin/env python3
tools/review:#!/usr/bin/env python
tools/run-mypy:#!/usr/bin/env python
tools/server_lib/lister.py:#!/usr/bin/env python
tools/test-bots:#!/usr/bin/env python3
tools/test-botserver:#!/usr/bin/env python
tools/test-lib:#!/usr/bin/env python3
tools/test-zulip:#!/usr/bin/env python
```

Reply:
Not sure what you mean? Do we have a rule against using python3? It I feel like since `tools/provision` uses python3, all of the other scripts should use python3 as well, since the virtualenv that `tools/provision` creates is a python3 env. What do you think?

Reply:
This does make sense, particularly given that all but the `zulip/` component should be python3 now? That said, there may be a separate recommendation in terms of this use.